tsconfig.json

```diff
@@ -15,7 +15,8 @@
     "noImplicitReturns": true,
     "noImplicitThis": true,
     "noImplicitAny": true,
-    "strictNullChecks": true
+    "strictNullChecks": true,
+    "allowSyntheticDefaultImports": true
```

Comment:
Hvorfor er denne lagt på?

Reply:
For å få en smudere import. Da slipper vi å måtte `require('dayjs')`, og kan heller import på vanligvis

Reply:
Vi har jo ikke trengt `require` noen andre steder, hvorfor er det annerledes med dayjs?

Reply:
Require bruker kun i nodejs